public/v4/apps/corteza.yml

```diff
@@ -0,0 +1,63 @@
+captainVersion: 4
+
+services:
+    $$cap_appname:
+        image: cortezaproject/corteza:$$corteza_version
+        restart: always
+        depends_on:
+            - $$cap_appname-db
+        volumes:
+            - $$cap_appname-data:/data"
+        environment:
+            VIRTUAL_HOST: $$cap_appname.$$cap_root_domain
+            DB_DSN: postgres://$$cap_postgre_username:$$cap_postgre_password@srv-captain--$$cap_appname-db:5432/$$cap_postgre_db_name?sslmode=disable
+            HTTP_WEBAPP_ENABLED: true
+            DOMAIN: $$cap_appname.$$cap_root_domain
+            VERSION: $$corteza_version
+    $$cap_appname-db:
+        image: postgres:13
```

Comment:
This app needs persistent volume, otherwise after a restart all data will be lost.